mathesar_ui/src/component-library/tabs/TabContainer.svelte

```diff
@@ -24,6 +24,7 @@
   export let fillTabWidth = false;
   export let fillContainerHeight = false;
   export let uniformTabWidth = true;
+  export let tabStyle = 'default';
```

Comment:
It would be nice to specify a union type like `'default' | 'compact'` (or similar) here, so that TS doesn't infer `string`.